component/vshn_postgres.jsonnet

```diff
@@ -344,14 +344,16 @@ local appcatFuncRole = kube.Role('appcat-function:stackgres-restapi-admin') {
      '20_role_vshn_postgresrestore': [ restoreRole, restoreServiceAccount, restoreClusterRoleBinding ],
      '20_plans_vshn_postgresql': plansCM,
      '21_composition_vshn_postgres': composition,
-     '22_appcat_func_role': appcatFuncRole,
```

Comment:
This was wrongly deployed to the control-plane. The control plane doesn't have stackgres installed, so all these bindings will fail.